package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "serve": "vue-cli-service serve --mode development",
     "serve:prod": "yarn run build:prod && http-server ./dist",
-    "build:prod": "yarn run generate-locales && rimraf ./dist && vue-cli-service build",
+    "build:prod": "yarn run generate-locales && rimraf --glob ./dist && vue-cli-service build",
```

Comment:
What's the reason for the command change here?

Reply:
Reason: c&p from my other repo, but not required in this case 👍